x-pack/plugins/security_solution/public/cases/links.ts

```diff
@@ -16,16 +23,29 @@ export const getCasesLinkItems = (): LinkItem => {
       [SecurityPageName.case]: {
         globalNavEnabled: true,
         globalNavOrder: 5,
-        capabilities: [`${CASES_FEATURE_ID}.read_cases`],
+        capabilities: [`${CASES_FEATURE_ID}.${READ_CASES_CAPABILITY}`],
       },
       [SecurityPageName.caseConfigure]: {
-        capabilities: [`${CASES_FEATURE_ID}.crud_cases`],
+        capabilities: [
+          `${CASES_FEATURE_ID}.${CREATE_CASES_CAPABILITY}`,
+          `${CASES_FEATURE_ID}.${READ_CASES_CAPABILITY}`,
+          `${CASES_FEATURE_ID}.${UPDATE_CASES_CAPABILITY}`,
+          `${CASES_FEATURE_ID}.${DELETE_CASES_CAPABILITY}`,
+          `${CASES_FEATURE_ID}.${PUSH_CASES_CAPABILITY}`,
+        ],
         licenseType: 'gold',
         sideNavDisabled: true,
         hideTimeline: true,
       },
+      // TODO: can we just use create here?
       [SecurityPageName.caseCreate]: {
-        capabilities: [`${CASES_FEATURE_ID}.crud_cases`],
+        capabilities: [
```

Comment:
@cnasikas same question as above here, should this just be `create`?

Reply:
We decided to only split out deletion specific things, so since this was all, we'll keep it that way.

Reply:
same reminder as above 😊